include/spirv-tools/optimizer.hpp

```diff
@@ -376,18 +376,6 @@ Optimizer::PassToken CreateCommonUniformElimPass();
 // eliminated with standard dead code elimination.
 Optimizer::PassToken CreateAggressiveDCEPass();
 
-// Creates a pass to consolidate uniform references.
```

Comment:
Not a mistake. This was an extra declaration that somehow snuck in. I piggybacked it into this PR rather than create a new PR for something so minor.